examples/03_connectivity/plot_group_level_connectivity.py

```diff
@@ -0,0 +1,222 @@
+
+###############################################################################
+# Look at blocks structure, reflecting functional networks.
+
+###############################################################################
+# Examine partial correlations
+# ----------------------------
+# We can also study **direct connections**, revealed by partial correlation
+# coefficients. We just change the `ConnectivityMeasure` kind
+partial_correlation_measure = ConnectivityMeasure(kind='partial correlation')
+
+###############################################################################
+# and repeat the previous operation.
+partial_correlation_matrices = partial_correlation_measure.fit_transform(
+    adhd_subjects)
+
+###############################################################################
+# Most of direct connections are weaker than full connections, resulting
+# in a sparse mean connectome graph.
+plot_matrices(partial_correlation_matrices[:4], 'partial')
+plotting.plot_connectome(
+    partial_correlation_measure.mean_, msdl_coords,
+    title='mean partial correlation over 13 ADHD subjects')
+
+###############################################################################
+# Extract subjects variabilities around a robust group connectivity
+# -----------------------------------------------------------------
+
+###############################################################################
+# We can use **both** correlations and partial correlations to capture
+# reproducible connectivity patterns at the group-level and build a **robust**
+# **group connectivity matrix**. This is done by the **tangent** kind.
+tangent_measure = ConnectivityMeasure(kind='tangent')
+
+###############################################################################
+# We fit our ADHD group and get the group connectivity matrix stored as
+# in `tangent_measure.mean_`, and individual deviation matrices of each subject
+# from it.
+tangent_matrices = tangent_measure.fit_transform(adhd_subjects)
+
+###############################################################################
+# `tangent_matrices` model individual connectivities as
+# **perturbations** of the group connectivity matrix `tangent_measure.mean_`.
+# Keep in mind that these subjects-to-group variability matrices do not
+# straight reflect individual brain connections. For instance negative
+# coefficients can not be interpreted as anticorrelated regions.
+plot_matrices(tangent_matrices[:4], 'tangent variability')
+plotting.plot_connectome(
+    tangent_measure.mean_, msdl_coords,
+    title='mean tangent connectivity over 13 ADHD subjects')
+
+###############################################################################
+# The mean connectome graph is not as sparse as partial correlations graph,
+# yet it is less dense than correlations graph.
+
+###############################################################################
+# Which connectivity kind is more powerful? A classification answer
```

Comment:
As a title here, I would suggest "What kind of connectivity is most powerful for classification?"